spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/vertexai/gemini/VertexAiGeminiAutoConfiguration.java

```diff
@@ -74,6 +77,8 @@ public VertexAI vertexAi(VertexAiGeminiConnectionProperties connectionProperties
 
 	@Bean
 	@ConditionalOnMissingBean
+	@ConditionalOnProperty(prefix = VertexAiGeminiChatProperties.CONFIG_PREFIX, name = "enabled", havingValue = "true",
+			matchIfMissing = true)
```

Comment:
Because, currently,  we are using a single boot starter for all Bedrock providers, the Bedrock models should be [disabled by default](https://docs.spring.io/spring-ai/reference/api/bedrock-chat.html#_enable_selected_bedrock_model).